### DISCUSSION TITLE
The same People edits by admin to show for all users?

### DISCUSSION THREAD

**Discussion by spectromas** [nextcloud/recognize#810] [Q&A] [answered]
I'm not sure if this is a Memories issue, a Recognize issue, or expected behaviour, but I'm not sure how to solve the problem. As the admin I have named, merged, and removed People in Memories that Recognise picked up but these changes are not showing for other users, People just appear as they originally did without the edits for them.

I'm really hoping there is a way around this since I really don't want users to each have to edit the People section to make it usable.

**Comment by marcelklehr** [comment, marked as answer]
> I really don't want users to each have to edit the People section

This is how it is designed to be. Currently face clusters are not shared between users.
